Cargo.toml

```diff
@@ -4,3 +4,12 @@ version = "0.1.0"
 edition = "2024"
 
 [dependencies]
+
+[lints.rust]
+unsafe_code = "forbid"
+
+[lints.clippy]
+unwrap_used = "warn"
+enum_glob_use = { level = "deny", priority = 1 }
+pedantic = { level = "deny", priority = -1 }
+nursery = { level = "deny", priority = -1 }
```

Comment:
These are, frankly, very extreme and annoying, with loads of false positives. Once you understand a lint, by all means add individual `ignore` lines in code that is actually fine. But you must learn the rules first!

Reply:
Heck yes 🤘